### DISCUSSION TITLE
Better wording when asking for Borg repo passphrase

### DISCUSSION THREAD

**Discussion by hutson** [borgbase/vorta#835] [Ideas]
**Is your feature request related to a problem? Please describe.**

I was frustrated when it was not intuitive what the _Borg passphrase_ input field expected.

Since I just started exploring Vorta, along with BorgBase for hosting, I was a little confused by the phrase.

Was I supposed to provide my BorgBase password? Or was the _passphrase_ used to enable local encryption?

I went searching and eventually came across https://github.com/borgbase/vorta/issues/309. After reading through the link provided by Thomas, I realized it must mean the local encryption passphrase. 

**Describe the solution you'd like**

A tooltip next to _Borg passphrase_ that provides in-context help to users not aware of Borg or its terminology.

**Describe alternatives you've considered**

Telling people to read the manual before using Vorta. :man_shrugging:

**Comment by m3nu** [comment]
Yeah, we could be more clear here.

Since Vorta aims to be provider-agnostic, it will never ask for any BorgBase data.

**Comment by ThomasWaldmann** [comment]
It's the `borg repository key passphrase`.

The key material is encrypted with a key derived from this passphrase.
The use the key, one needs to decrypt / "open" it by giving the passphrase.

**Comment by ThomasWaldmann** [comment]
Hmm, maybe better `borg key passphrase` or people will confuse it with `repokey` although this is not meant here.

**Comment by samuel-w** [comment]
How about this:
![wordingh](https://user-images.githubusercontent.com/67625312/86430458-f396ba00-bcb7-11ea-801b-87f6710204a1.png)

**Comment by hutson** [comment]
@samuel-w maybe. It's a little difficult to know whether it makes sense given that I've been using Vorta for a little while now. So I ask myself, would other users think `Repository passphrase` is the password used to unlock their SSH key when Borg connects to the remote repository? Or, again, the password for the Borg server provider that provides the repository?

Maybe I'm coming at this with the assumption that the prompt is asking me to fill out a passphrase that must exist somewhere else. It's like a login form. You provide the username and password for the site you are attempting to log into. You don't provide a _new_ passphrase at login.

Anyways, that's my two cents. :smiley:

**Comment by m3nu** [comment]
Not easy. Maybe "Borg Repo Passphrase", as Thomas suggested? Qt also supports adding small text below the field. This could say "Used to encrypt your Borg repository key. Not related to SSH authentication". 🤔

**Comment by samuel-w** [comment]
We could also do it 
![uiOpt2](https://user-images.githubusercontent.com/67625312/89704482-a098ce00-d919-11ea-81a5-49e9de390d2c.png)
through a tooltip
![uiOpt1](https://user-images.githubusercontent.com/67625312/89704479-9bd41a00-d919-11ea-8724-af1ecbfc1cbf.png)
or a placeholder.
Maybe the tooltip should be on the "Borg passphrase:".
I think we should do it with a placeholder, since the tooltip requires a mouseover interaction. Thoughts?

**Comment by ThomasWaldmann** [comment]
Maybe label should be `Borg key passphrase`.

Also it should not say `used to encrypt ... key` for an `existing repository`.

If the repo exists, we already have the key and we `decrypt` it with that passphrase.

Also not sure that we should ever say what something is **not** related to if we can just clearly say what it is.
